Sources/MapboxMaps/Foundation/MapView.swift

```diff
@@ -34,6 +35,17 @@ open class MapView: UIView {
     /// Controls the addition/removal of annotations to the map.
     public internal(set) var annotations: AnnotationOrchestrator!
 
+    /// Property that describes the current language for `SymbolLayer.textField`
+    public var locale: Locale = Locale.current {
```

Comment:
`Locale.current` only matches the language that the iOS UI is currently displayed in, which can differ from both the surrounding application and the user’s preference. `Bundle.preferredLocalizations` has the advantage of matching the language that the rest of the application is currently displayed in. `Locale.preferredLanguages` has the advantage of matching the user’s preferred content language settings, even if the application is unavailable in the preferred language.